pyproject.toml

```diff
@@ -46,6 +46,7 @@ select = [
   "ICN",  # flake8-import-conventions
   "INT",  # flake8-gettext
   "PL",   # Pylint
+  "PLR",  # Pylint
```

Comment:
`PL` includes all of `PLC`, `PLE`, `PLR`, and `PLW` so this change can be reverted.